### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by edakturk14**
Hey Carlos, here's the pr for issue https://github.com/scaffold-eth/se-2/issues/67

I added a placeholder for the color switch icon. Here's how it looks:

Screen Shot 2022-12-16 at 16 47 33

Thanks!

### CODE REVIEW

File: packages/nextjs/components/Footer.tsx [hunk 1]
```diff
@@ -9,13 +9,40 @@ export default function Footer() {
...
+              </a>
+            </div>
+            <div>
+              Built with 💖 at 🏰{" "}
+              <a href="https://buidlguidl.com/" target="_blank" rel="noreferrer">
+                BuidlGuidl
+              </a>
+            </div>
+            <div>
+              <a href="https://google.com/" target="_blank" rel="noreferrer">
```
Comment (damianmarti) [inline packages/nextjs/components/Footer.tsx:36]: I think that the link here should be the link to the Telegram group https://t.me/joinchat/KByvmRe5wkR-8F_zz6AjpA

Carlos commented it at the related issue https://github.com/scaffold-eth/se-2/issues/67

Comment (edakturk14) [reply packages/nextjs/components/Footer.tsx]: oh my bad, didn't realize the link there. Updated https://github.com/scaffold-eth/se-2/pull/75/commits/158ecbbfd473e05e5a38e6e2ca5828db988b4643

Thank you Damu!!

**Review by damianmarti — APPROVED** [review]

**Comment by edakturk14** [comment]
Hey Carlos & Damu,

As discussed w/Carlos, I have made some edits:
- Remove the navbar, since this is the footer 
- Add the line below the links (fork me, built with <3 at BG, Support)
- Add the heart icon 
- Add span in-between the center menu items.

here's how it looks now:
Screen Shot 2022-12-16 at 19 24 28

**Comment by carletex** [comment]
Looks great @edakturk14 

Just made a tiny tweak to fix the spacing caused by flex.

**Before**
image


**After**
image

**Pull request merged**